.prettierignore

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
[Prettier](https://prettier.io/) helps keep your code base looking neat. It helps with styling. This config tells prettier not to worry about styling these files